pyvalid/validators/__number.py

```diff
@@ -47,13 +67,16 @@ def checkers(self):
         in_range=[Iterable, Container], not_in_range=[Iterable, Container]
     )
     def __init__(self, **kwargs):
+        number_type = kwargs.get('number_type', None)
         min_val = kwargs.get('min_val', None)
         max_val = kwargs.get('max_val', None)
         if min_val is not None and max_val is not None and min_val > max_val:
             raise ValueError('Min value can\'t be greater than max value!')
         in_range = kwargs.get('in_range', None)
         not_in_range = kwargs.get('not_in_range', None)
+
```

Comment:
There should be only 1 empty line, please don't add the second one
Thanks!

Reply:
Hi @uzumaxy There was no empty line before, so added one as it was looking crowded. Let me know if I should still remove it. Thanks.

Reply:
Got it, thanks!